package.json

```diff
@@ -40,8 +40,8 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "@storybook/addon-info": "^3.2.0",
-    "@storybook/react": "^3.2.3",
+    "@storybook/addon-info": "^3.2.17",
```

Comment:
Do we need the package updates? If not, I would prefer having these on a different PR.